### PR TITLE
Refactor some code

### PR DESCRIPTION
**Pull request by onee-only**

### CODE REVIEW

**Comment by onee-only** [comment]
Sorry, forgot to test it.

File: src/server.go [hunk 1]
```diff
@@ -236,17 +236,15 @@ func parseGetParams(query string) getParams {
...
-				if err == nil {
-					params.limit = val
-				}
+				params.limit = val
 			case "offset":
-				val, err := strconv.Atoi(parts[1])
-				if err == nil {
-					params.offset = val
-				}
+				params.offset = val
```
Comment (junegunn) [inline src/server.go:247]: I'm not sure about this. We don't want to call `Atoi` for other unsupported parameters.

This is what I'm suggesting.
```go
switch parts[0] {
case "limit", "offset":
	if val, err := strconv.Atoi(parts[1]); err == nil {
		if parts[0] == "limit" {
			params.limit = val
		} else {
			params.offset = val
		}
	}
}
```

Comment (onee-only) [reply src/server.go:247]: This looks much better! I'll update it as you suggested. Thank you for your feedback!

**Comment by junegunn** [comment]
Merged, thanks!

**Pull request merged**